app/src/test/resources/testcode-cpp/geometry.h

```diff
@@ -0,0 +1,25 @@
+#ifndef GEOMETRY_H
+#define GEOMETRY_H
+
+namespace shapes {
```

Comment:
Could we add a test for anonymous namespaces? Just to ensure the parser doesn't always expect a name, e.g.
```c++
// Anonymous namespace
namespace {
    int value = 10;
}
```